### ISSUE TITLE
🐛 [BUG] - Add "aria-label" in Some Files to improve accessibility (for screen readers)

### ISSUE THREAD

**Issue by NarendraKoya999** [ZeroOctave/resource-gallery#273]
## 🐛 Bug Report Form

|                |                     |
|----------------|---------------------|
| **Description** |  Add `"aria-label"` in Some Files like` contctform.astro and footer.astro` to improve accessibility (for screen readers)  |
| **Reproduction URL** | https://resourcegallery.live/|
| **Reproduction Steps** | 1. Go To https://resourcegallery.live/<br>2. Observe in `social icons when hovering effects done  `in footer Section<br>3. There is no` screen reading available on respective social Icons`<br>4. So add `aria-labels` in respective above mentioned Files|
| **Screenshots** | ![Screenshot 1](screenshot1.png) |
| **Logs** | Relevant log output goes here |
| **Additional Context** | `aria-label` can be used in cases where text that could label the element is not visible. Also, `aria-label` is allowed on any element that can have an accessible name, in practice, `aria-label` is only supported on interactive elements, widgets, landmarks, images and iframes. |

🌐 **Browsers**
Which browsers are you experiencing the problem on? (Select all that apply)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Opera
- [ ] Brave

💻 **OS**
What is the impacted environment (operating system)? (Select all that apply)

- [x] Windows
- [ ] Linux
- [ ] Mac
- [ ] iOS
- [ ] Android

📝 **Sections Completed**
Check the box of the sections you have completed:

- [x] Description
- [x] Reproduction URL
- [x] Reproduction Steps
- [ ] Screenshots
- [ ] Logs
- [x] Browsers
- [x] OS
- [x] Additional Context

🔍 **Priority**
Please select the priority of the bug:
- [x] High
- [ ] Medium
- [ ] Low

📜 **Code of Conduct**
By submitting this issue, you agree to follow our Code of Conduct:

- [x] I follow Contributing Guidelines of this project.
- [x] I have checked the current issues for duplicate problems.
- [x] I have read the readme.

**Comment by NarendraKoya999** [comment]
Hi @Astrodevil and @Alkaison . Please Review on this ISSUE as soon as Possible. Thank You !

**Comment by Alkaison** [comment]
@NarendraKoya999 assigned ✅
